src/components/UI/Charts/Sunburst/Sunburst.tsx

```diff
@@ -41,12 +69,18 @@ class Sunburst extends Component<SunburstProps, State> {
 
   state = {
     selectedPhenotypeInfo: pickPhenotypeInfo(this.props.data),
+    phenotypeTree: [],
   };
 
-  getSelectedPhenotype = (phenotype: Phenotype) =>
+  getSelectedPhenotype = (phenotype: Phenotype) => {
+    const phenotypes = phenotype.key.match(RegexExtractPhenotype);
+    const phenoReversed = (phenotypes || []).reverse();
```

Comment:
Note: reverse() modifies the array. It can be ok. Just wanna let you know in case it's not.